apps/protocol-api/src/main.ts

```diff
@@ -80,6 +81,7 @@ const permissions = shield(
       createUserContribution: and(OwnsData, isAuthenticated),
       createUserCustom: or(hasToken, and(OwnsData, isAuthenticated)),
       createUserOnChainAttestation: isAuthenticated,
+      deleteContribution: or(OwnsData, isAuthenticated),
```

Comment:
We are going to need a custom resolver that checks the user address owns the contribution they are deleting. Right now any logged in user can delete any contribution